OBSDOCS-71: [POC] Observability operators upgrade resource (#80691)
openshift/openshift-docs: observability/index.adoc

* {PM-shortname-c}
endif::openshift-dedicated,openshift-rosa[]

{ObservabilityLongName} connects open-source observability tools and technologies to create a unified {ObservabilityShortName} solution. The components of {ObservabilityLongName} work together to help you collect, store, deliver, analyze, and visualize data.

[NOTE]
====

> With the exception of monitoring, {ObservabilityLongName} components have distinct release cycles separate from the core {product-name}. The Red{nbsp}Hat link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] page outlines their release compatibility.

Reviewer: what's the value of `product-name`?
from https://80691--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/index.html
```
With the exception of monitoring, Red Hat OpenShift Observability components have distinct release cycles separate from the core {product-name}. The Red Hat [OpenShift Operator Life Cycles](https://access.redhat.com/support/policy/updates/openshift_operators) page outlines their release compatibility.
```
`product-name` is not replaced with a actual name

Author: That is a mistake of mine :cry: , thank you for noticing! Should be product-title. I quickly fixed it.

> With the exception of monitoring, {ObservabilityLongName} components have distinct release cycles separate from the core {product-title}. The Red{nbsp}Hat link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] page outlines their release compatibility.

Reviewer 2: ```suggestion
With the exception of monitoring, {ObservabilityLongName} components have distinct release cycles that are separate from the core {product-title} release cycles. See link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] (Red{nbsp}Hat Customer Portal).
```

Reviewer 2: I assume the following notice does not impact the public availability of the page?

![Screenshot from 2024-08-27 12-51-09](https://github.com/user-attachments/assets/c1ee1234-8a41-4136-8906-06ba3db85b44)

Author: Thank you for the review! Regarding the suggestion, I will rewrite it as:
"With the exception of monitoring, {ObservabilityLongName} components have distinct release cycles separate from the core {product-title} release cycles. See the Red{nbsp}Hat link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] page."

* I will omit the suggestion of "that are" to avoid redundancy and retain minimalism.
* I will not include the brackets as, according to IBM, we shoule prefer to avoid the brackets.

Regarding the link, yes, only the note is private and does not impact public availibility :+1:

Reviewer 2: Thanks, @eromanova97 . I suggested the brackets to give a slight hint to the customer that they will be leaving the document to go to another part of the Customer Portal.

> With the exception of monitoring, {ObservabilityLongName} components have distinct release cycles separate from the core {product-title} release cycles. See the Red{nbsp}Hat link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] page for their release compatibility.

Reviewer 3: ```suggestion
Except for monitoring, {ObservabilityLongName} components have distinct release cycles separate from the core {product-title} release cycles. See the Red{nbsp}Hat link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] page for their release compatibility.
```

Author: Thank you @Srivaralakshmi for the comment! "except for" sounded informal to me here, so I chose to use "with the exception of". In the RH style guide, I found instances of using both, so I would prefer to leave the more formal version, if that is okay with you. Thank you!

====

> 

Reviewer 3: L33 Please refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-subsections-level-1-and-lower-headings

For any Level 1 or lower subsections in an assembly (for example, ==, ===, etc.), add a _{context} variable to the end of anchor IDs

Author: Thank you for noticing! I added the variable :+1:
